lib/react-native/add-content-type.js

```diff
@@ -0,0 +1,15 @@
+function addContentType(req) {
+  var httpRequest = req.httpRequest || {};
+  var headers = httpRequest.headers;
+  // We don't want to force a content type on presigned urls
+  if (headers && !req.isPresigned()) {
+    if (httpRequest.body && !headers['Content-Type']) {
+      // React Native's android XHR requires Content-Type to be defined if there is a body
```

Comment:
Not a blocker, but it would be worthwhile to open an issue on React Native or comment on an existing one. (This looks related to https://github.com/facebook/react-native/issues/10362, but there might be another one that's still open.)